podcasts/EpisodesDataManager.swift

```diff
@@ -0,0 +1,127 @@
+import PocketCastsDataModel
+import PocketCastsServer
+import DifferenceKit
+
+/// Returns a list of episodes for an specific section
+/// Depending on the section, it returns a DifferenceKit ArraySection
```

Comment:
This seems like it should be above the episodes for podcast. 

```suggestion
/// Returns a podcasts episodes that are grouped by `PodcastGrouping` 
/// Use `uuidsToFilter` to filter the episode UUIDs to only those in the array
```
Thoughts?

Reply:
My intention was for it to be the description of the class itself, section being a "part of the app". But now I see how confusing that is, moved to the podcast method.